lib/tasks/recovery.rake

```diff
@@ -0,0 +1,6 @@
+namespace :recovery do
+  desc "Requeue content changes and messages over one-hour old for which jobs have been lost"
+  task recover_lost_jobs: :environment do
+    RecoverLostJobsWorker.perform_async
+  end
+end
```

Comment:
Do we need this task? I'm not sure under what scenarios we'd run it. It seems to be we'd need it if the sidekiq scheduling failed but if that happened we'd probably not trust putting another thing on sidekiq.

Anyway we should probably have a basic test for this too if this is something that adds value.

Reply:
@benthorner the card suggest a rake task, but this is a fair point indeed. What is your opinion on this?

Reply:
It's referenced in the docs, and I think the reasoning still stands. However, the card could have been more specific: if someone on 2ndline is running this, it's probably because the work needs requeueing immediately (not after ~1.25 hours). Really, it depends on what the new documentation is going to look like: we auto-fix this alert for you, but if that doesn't work .... (we need some kind of debugging steps, although maybe a rake task isn't one of them).

Reply:
Makes sense. I've tried putting myself in the perspective of what I'd do if I saw the alert and tried to jot it down, maybe this will help the documentation.

1. Check Sentry and Kibana for any indication either this job is raising errors and failing or if the ProcessContentChangeWorker is raising errors (this seems most likely scenario)
2. Identify which changes are not processed: `ContentChange.unprocessed.where("created_at < ?", 2.hours.ago)`
3. Try run the process manually from a console and see if anything breaks. For example: `ContentChange.unprocessed.where("created_at < ?", 2.hours.ago).pluck(:id).map { |id| ProcessContentChangeWorker.new.perform(id) }`

Since the [documentation](https://docs.publishing.service.gov.uk/manual/alerts/email-alert-api-unprocessed-content-changes.html) is already rather focussed on manual console tasks it's probably worth just amending things to this list rather than the rake task.

I'd suggest adding some info on how to check if a job raised an error as the main thing then having bits on how to find the content changes and how to process them (which seems to already be there but could use some rephrasing.